setup.py

```diff
@@ -55,9 +55,7 @@ def get_readme() -> str:
         "stable-baselines~=2.10.0",
         "jax~=0.1.66",
         "jaxlib~=0.1.47",
-        # sacred==0.7.5 build is broken without pymongo
-        # sacred>0.7.4 have non-picklable config objects (see GH #109)
-        "sacred==0.7.4",
+        "sacred",
```

Comment:
Should we lock to 0.8.1?

Reply:
I'd rather not -- you're then forcing users of `imitation` to choose a particular `sacred` version. You probably want to have a minimum version though?